include/tvm/meta_schedule/integration.h

```diff
@@ -215,6 +215,9 @@ class ApplyHistoryBest : public MetaScheduleContext {
                                                     ApplyHistoryBestNode);
 };
 
+Optional<ObjectRef> ContextQueryInsideWithScope(runtime::String task_name, IRModule mod,
+                                                Target target,
+                                                Optional<Array<IRModule>> dispatched);
```

Comment:
Instead of creating a new global method, let's directly call `MetaScheduleContext::QueryInsideWithScope`